src/passthrough/mod.rs

```diff
@@ -777,7 +777,7 @@ impl<S: BitmapSlice + Send + Sync> PassthroughFs<S> {
                         // We just removed the last refcount for this inode.
                         // The allocated inode number should be kept in the map when use_host_ino
                         // is false or inode is bigger than MAX_HOST_INO.
```

Comment:
Update the comment as well? "inode" should refer to host inode or real inode?

Reply:
fixed